docs-v2/content/en/docs/workflows/handling-platforms.md

```diff
@@ -0,0 +1,230 @@
+  ...
+  Building [skaffold-example]...
+  Target platforms: [linux/amd64,linux/arm64]
+  ...
+  [+] Building 0.3s (13/13) FINISHED
+  ...
+  => => writing image sha256:10af3142e460566f5791c48758f0040cef6932cbcb0766082dcbb0d8db7653e7
+  => => naming to gcr.io/k8s-skaffold/skaffold-example:latest_linux_amd64
+  ...
+  latest_linux_amd64: digest: sha256:15bd4f2380e99b3563f8add1aba9691e414d4cc5701363d9c74960a20fb276c4 size: 739
+  ...
+  [+] Building 52.8s (13/13) FINISHED
+  ...
+  => => writing image sha256:68866691e2d6f079b116e097ae4e67a53eaf89e825b52d6f31f2e9cc566974de
+  => => naming to gcr.io/k8s-skaffold/skaffold-example:latest_linux_arm64
+  ...
+  4e0c2525c370: Pushed
+  latest_linux_arm64: digest: sha256:868d0aec1cc7d2ed1fa1e840f38ff1aa50c3cc3d3232ea17a065618eaec4e82b size: 739
+  Build [skaffold-example] succeeded
+  ```
+
+* Validate that the image just built was multi-arch, by running the following `docker` command:
+
+  ```cmd
+  docker manifest inspect your/container/registry/skaffold-example:latest | grep -A 3 "platform"
+  ```
+
+  Outputs:
+
+  ```cmd
+    "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+    }
+  --
+    "platform": {
+        "architecture": "arm64",
+        "os": "linux"
+    }
+  ```
+
+* Now if we render the Kubernetes Pod manifest for this multi-arch image, then it'll have platform affinity definition targeting both `linux/amd64` and `linux/arm64` architectures.
+
+  ```cmd
+  skaffold render --default-repo=your/container/registry --enable-platform-node-affinity
+  ```
+  
+  Outputs:
+
+  ```cmd
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: getting-started
+    namespace: default
+  spec:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.io/os
+              operator: In
+              values:
+              - linux
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+              - amd64
+          - matchExpressions:
+            - key: kubernetes.io/os
+              operator: In
+              values:
+              - linux
+            - key: kubernetes.io/arch
+              operator: In
+              values:
+              - arm64
+    containers:
+    - image: gcr.io/k8s-skaffold/skaffold-example:latest@sha256:9ecf4e52f7ff64b35deacf9d6eedc03f35d69e0b4bf3679b97ba492f4389f784
+      name: getting-started
+  ```
+
+{{< alert title="Note" >}}
+
+* Multi-arch images need to be pushed to a container registry, as the local Docker deamon doesn't yet support storing multi-arch images.
+
+* For interactive modes like `skaffold dev` and `skaffold debug` requiring fast and repeated `build-render-deploy` iterations, Skaffold will choose only one build architecture and build a single-platform image, even if you specify multiple target platforms.
```

Comment:
nit: Does it make sense to add a warning that skaffold outputs stating this?

Reply:
we have https://github.com/GoogleContainerTools/skaffold/issues/7633 open for that. I'll also include the second point as a warning as part of that task.